kodi_addon_checker/check_files.py

```diff
@@ -89,6 +90,18 @@ def addon_xml_matches_folder(report: Report, addon_path: str, parsed_xml, folder
             report.add(Record(PROBLEM, "Addon id and folder name does not match."))
 
 
+def check_for_legacy_changelog(report: Report, addon_path: str, branch: str):
+    """Check whehter a legacy changelog.txt is present in the addon folder root"""
+    changelog_path = os.path.join(addon_path, "changelog.txt")
+    if os.path.isfile(changelog_path):
+        if KodiVersion(branch) < KodiVersion("leia"):
```

Comment:
please use a single line:

```python
report.add(
    Record(
        ERROR if KodiVersion(branch) >= KodiVersion("leia") else WARNING
       "A changelog.txt file was found. Use the <news> tag in addon.xml "
                                       "for the most recent changes instead of a changelog.txt."))
```

Reply:
Maybe point to the wiki link for `<news>` in the message so that users know what to do

Reply:
Done